### DISCUSSION TITLE
BUG fieldservice_calendar

### DISCUSSION THREAD

**Discussion by MarcosGraficon** [OCA/odoo-community.org#145] [Q&A]
I found a bug that if you create a "ticket" already related to a worker, in the meeting he doesnt appear, u need to reasign him to make him appear, any ideas how to fix i think maybe the create is bugged and the writhe method works fine

**Comment by vvrossem** [comment]
Hello, could you please create an issue in the related repository (https://github.com/OCA/field-service/issues), explaining how to reproduce this bug? Thank you
